app/MMB/src/navigation/NavigationStack.js

```diff
@@ -42,6 +43,22 @@ const TravelDocumentStack = StackNavigator(
   },
 );
 
+// This is not good enough since it just opens as a new page, not modal
+// It is complicated, needs to be improved later
```

Comment:
We would need to create an issue to make the process of opening windows as modals a lot easier.